src/components/DisplayArticles.jsx

```diff
@@ -18,7 +18,7 @@ const DisplayArticles = props => {
   }
   let articleDisplay = articles.map(article => {
     return (
-      <Grid key={article.id} align="center">
+      <Grid key={article.id} id={`article-${article.id}`} align="center">
```

Comment:
When we're refactoring, we could try to wrap each article in a grid column instead of a grid and column. 
But we need to add article-id to the column if we do. 
`<Grid.Column key={article.id} id={`article-${article.id}`}>`

Reply:
we only need to have ONE grid, and that's on row 38 like this
`return <Grid align="center">{articleDisplay}</Grid>;`

Reply:
I tried making the updates as you suggested here but the formatting looked really bizarre kinda smooshing all the articles into thin strips, and I wasn't sure how to quickly resolve the formatting issue. I can show you when you get on, but I think for now our it might be best to move forward with the current format.

Reply:
I agree. For now we should keep this and if we have the time later we can adress it.